### ISSUE TITLE
I want to filter reports based upon brands and categories, which certainly now is not possible, I guess, can you please find an alternative for it.

### ISSUE THREAD

**Issue by mtalhadev** [uilibrary/Stocky-Issues-and-Feature-request#17]
Right now the reports have alot of issues.
We can't filter reports based on products, categories, brands.
Like today how much of which product/category/brand is sold.

**Comment by naime-hossain** [comment]
Thank you for the suggestions, we will add all the features requested by our buyer :) you can rate our item on ThemeForest which will motivate our team toward further development :)

**Comment by mtalhadev** [comment]
I am very glad to see such development on suggestions. I wish you the best of luck. If you want I can tell you the whole QA for it, I am a senior saas manager, would tell you the detailed bugs where this template lacks as of the realistic ground needs.

**Comment by naime-hossain** [comment]
Yes you can give us full QA , that will be awesome . if you like our work, you can leave a review that will motivate our team toward further development if possible. If you don’t know how to provide ratings & review please visit this link: https://help.market.envato.com/hc/en-us/articles/203269490-Giving-Feedback-On-Items thank you
